### DISCUSSION TITLE
Minetest game support?

### DISCUSSION THREAD

**Discussion by sherkhan30452** [guillaumechereau/goxel#375] [Q&A]
Hi, I wanted to ask whether goxel supports Minetest? Can I make simple schematics in this program. If yes, how to import custom minetest textures into goxel.

**Comment by guillaumechereau** [comment]
There is basic import of minetest files.  No export at the moment.  Le me know if you find some bugs or limitations.

**Comment by Dolampo4ki** [comment]
Hello!
Tell me, is version 0.15.1 for a computer working? For me - neither .gox nor .vox files are loaded (saved). The working (stable) version is only 0.13.0

**Comment by guillaumechereau** [comment]
Hello @Dolampo4ki, are you on linux, mac or windows?

**Reply by Dolampo4ki** [reply]
windows 10

**Comment by guillaumechereau** [comment]
What happens when you try to save exactly?

**Comment by Dolampo4ki** [comment]
When selecting and loading a file, it does not appear on the screen (file -> import (open) -> Gox / Magica Voxel -> name file)
![2024-08-01_13-14-38](https://github.com/user-attachments/assets/e2ba8613-1854-4f97-bf2b-175f8f941445)
![2024-08-01_13-15-23](https://github.com/user-attachments/assets/b9b53986-e309-4cc3-acbc-fe2ba73c655d)
![2024-08-01_13-16-39](https://github.com/user-attachments/assets/f0140438-a04e-4a1d-9088-e3ea711e2e32)

**Comment by Dolampo4ki** [comment]
There is no collapse/close icon (like in older versions)
![2024-08-01_13-23-04](https://github.com/user-attachments/assets/e6ff36f4-d4f1-4cbc-b1c4-3ef44b9f27b1)

**Comment by guillaumechereau** [comment]
OK thanks, I'll try to test that later.  I rarely use windows so I usually release with the github build without even testing it!

**Comment by guillaumechereau** [comment]
I just tried on Windows 11 and it seemed to work.  By any change would you be able to build the app yourself in debug and run it in the terminal?
